test/Dialect/SV/hw-extract-test-code.mlir

```diff
@@ -132,3 +132,20 @@ module attributes {firrtl.extract.assert =  #hw.output_file<"dir3/", excludeFrom
     }
   }
 }
+
+// -----
+// Check "empty" modules are extracted
+
+// CHECK-LABEL: @WillBeEmpty(
+// CHECK-SAME: output_file = #hw.output_file<"testbench/", excludeFromFileList, includeReplicatedOps>
+module attributes {firrtl.extract.testbench = #hw.output_file<"testbench/", excludeFromFileList, includeReplicatedOps>} {
+  hw.module @WillBeEmpty(%clock: i1, %cond: i1) -> () {
+    sv.always posedge %clock  {
+      sv.cover %cond, immediate
+    }
+  }
```

Comment:
I was squinting at this test wondering why the PR wouldn't just use an empty module. However, this is specifically checking that after the cover is extracted, the module is then empty. It may be useful to include a comment for the test that this is what it is checking.

Reply:
Thanks for pointing it out, this is from when I had a more complex condition, and we actually needed to make sure such a module appeared "empty". Since we're just checking "input only" for now, I can simplify the test case.

Reply:
Oh, actually, this is here because there needs to be some "test code" to extract. Otherwise it gets skipped over.